packages/coq-serapi/coq-serapi.8.7.1+0.4.2/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer:   "e@x80.org"
+authors:      "Emilio Jesús Gallego Arias"
+homepage:     "https://github.com/ejgallego/coq-serapi"
+bug-reports:  "https://github.com/ejgallego/coq-serapi/issues"
+dev-repo:     "git+https://github.com/ejgallego/coq-serapi.git"
+license:      "GPL 3"
+
+name: "coq-serapi"
+available: [ ocaml-version >= "4.06.0" ]
+
+# ppx depends are so strict due to the issues with ppx_import and
+# ppx_driver integration in the past.
+depends: [
+  "coq"          { >= "8.7.1+1" & < "8.8" }
+  "camlp5"
+  "ocamlfind"    { build }
+  "ocamlbuild"   { build }
+  "ppx_import"   { build & >= "1.4" }
+  "ppx_deriving" { build & >= "4.2.1" }
+  "cmdliner"
+  "sexplib"
+  "ppx_driver"   { build & >= "v0.10.1" }
+  "ppx_sexp_conv"
```

Comment:
Shouldn't this be `{build}` as well?

Reply:
Indeed I had it corrected in the upstream repos but somehow I didn't propagate it correctly to opam-repository.